### PR TITLE
Create 12.pmw.hwc.csv

### PR DESCRIPTION
**Pull request by mucheinz**
time periods for circulation pump - english

### CODE REVIEW

File: ebusd-2.x.x/en/vaillant/12.pmw.hwc.csv [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# type (r[1-9];w;u),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment,,,
```
Comment (john30) [inline ebusd-2.x.x/en/vaillant/12.pmw.hwc.csv:1]: why not simply include timer.inc ?

File: ebusd-2.x.x/en/vaillant/12.pmw.hwc.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+#,PMW00,VPM-W Fresh water Station,PMW VPM-W,,,,,,,,,,,,,
+# ##### Generell #####,,,,,,,,,,,,,,,,
+# PMW: periods circulation pump,,,,,,,,,,,,,,,,
+*r,cp,,,,,"B504",,,,,,,,,,
```
Comment (john30) [inline ebusd-2.x.x/en/vaillant/12.pmw.hwc.csv:5]: this is no longer needed, right?

File: ebusd-2.x.x/en/vaillant/12.pmw.hwc.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+*w,cp,,,,,"B505","09",,,,,,,,,
```
Comment (john30) [inline ebusd-2.x.x/en/vaillant/12.pmw.hwc.csv:6]: this is no longer needed, right?

**Review by john30 — CHANGES_REQUESTED** [review]

**Review by mucheinz — COMMENTED** [review]
correct - no longer needed

**Pull request closed without merging**